boards/arm/cxd56xx/spresense/src/cxd56_bringup.c

```diff
@@ -139,6 +139,14 @@
  #endif
 #endif
 
+/****************************************************************************
+ * Public Data
+ ****************************************************************************/
+
+#ifdef CONFIG_VIDEO_ISX012
+FAR const struct video_devops_s *g_video_devops;
```

Comment:
should we put it to a common location?
drivers/video/video.c
It's very strange that the common code reference a symbole which is defined in board specific file.

Reply:
I thought about that, but then it could force video.o into the link even if it were not used.  I also though about a separate C file that contains nothing but the g_video_devops definition.

The real solution is to remove the global variable altogether.  It is very bad style for a driver interface.  For one thing, it limits the use of the driver to a single instance.  That is something that I have been careful to avoid in all drivers.  This breaks that rule.  See Issue #439.  Anything else is as band-aid to work around a very bad design.

Reply:
For now boards/arm/cxd56xx/spresense/src/cxd56_bringup.c  is the correct place because there and only there is can be conditioned on #ifdef CONFIG_VIDEO_ISX012 and only if it used.

So I am not going to change the code now.  It does need more work to be a respectable driver interface.  It sucks now in general and I cannot really make that better.

Reply:
Okay.. I did it anyway

Reply:
Okay.. I did change it as you asked.  I really hate working with crap designs like this.